src/PeriodEnd/Refunds/SFA.DAS.ProviderPayments.Calc.Refunds/Services/Dependencies/ILearnerProcessor.cs

```diff
@@ -0,0 +1,12 @@
+﻿
+using System.Collections.Generic;
+using SFA.DAS.ProviderPayments.Calc.Refunds.Dto;
+using SFA.DAS.ProviderPayments.Calc.Shared.Infrastructure.Data.Entities;
+
+namespace SFA.DAS.ProviderPayments.Calc.Refunds.Services.Dependencies
+{
+    public interface ILearnerProcessor
+    {
+        IEnumerable<RefundPaymentEntity> Process(LearnerData learnerData);
```

Comment:
There is no difference between a refund and a payment, and I would return a PaymentEntity here

Reply:
This wasn't clear in our design. I actually dislike returning Entities at all here, as these should be domain objects, but for the sake of simplicity  we could just return the Paymententity